yabai/.config/yabai/yabairc

```diff
@@ -40,6 +40,8 @@ yabai -m rule --add app="^Activity Monitor$" sticky=on layer=above manage=off
 yabai -m rule --add app="^Path Finder$" manage=off
 yabai -m rule --add app="Cato" sticky=on manage=off layer=above
 yabai -m rule --add app="Authy" sticky=on manage=off layer=above
+yabai -m rule --add title="Picture-in-Picture" sticky=on manage=off layer=above
+yabai -m rule --add app="^Archive Utility*" manage=off layer=above
```

Comment:
ok, now I have to try yabai...

Reply:
Actually this didn't work for me - if you get it to work let me know